README.md

```diff
@@ -9,7 +9,20 @@ In order to run this code, you'll need to install the Azure Functions [core tool
 
 Although not required, you may also want to install the Azure VS Code [extension.](https://code.visualstudio.com/docs/azure/extensions)
 
-### Local database setup
+### Run with Docker
+1. To build the containers, run the following command:  
+  `docker-compose build`  
+  You will need to re-run this command on any changes to the Dockerfile.
+  
+2. To initialize the database tables and test data, run:  
+  `docker-compose run --rm app yarn init-db`  
+  You'll only need to do this once.
+  
+3. To start the server run:  
+  `docker-compose run --rm app yarn start`
+  
+### Run without Docker
+#### Local database setup
```

Comment:
Should the createdb, schema and load data cmds be replaced with the `yarn init-db` cmd?

Reply:
Since I've updated the `init-db` script to use $PGDATABASE (docker env var), you can't actually use the script without having the env var set - I'll keep it as is.

Reply:
fwiw you can use a default value for an env var `PGDATABASE=${VARIABLE:-mapping_viz_local}`
https://stackoverflow.com/questions/2013547/assigning-default-values-to-shell-variables-with-a-single-command-in-bash

also if folks are running through docker-compose the env var should always be set by that compose file.